RNTester/Podfile.lock

```diff
@@ -11,108 +11,110 @@ PODS:
-  - React-RCTCameraRoll (1000.0.0):
-    - React-Core (= 1000.0.0)
-    - React-RCTImage (= 1000.0.0)
-  - React-RCTImage (1000.0.0):
-    - React-Core (= 1000.0.0)
-    - React-RCTNetwork (= 1000.0.0)
-  - React-RCTLinking (1000.0.0):
-    - React-Core (= 1000.0.0)
-  - React-RCTNetwork (1000.0.0):
-    - React-Core (= 1000.0.0)
-  - React-RCTPushNotification (1000.0.0):
-    - React-Core (= 1000.0.0)
-  - React-RCTSettings (1000.0.0):
-    - React-Core (= 1000.0.0)
-  - React-RCTText (1000.0.0):
-    - React-Core (= 1000.0.0)
-  - React-RCTVibration (1000.0.0):
-    - React-Core (= 1000.0.0)
-  - React-RCTWebSocket (1000.0.0):
-    - React-Core (= 1000.0.0)
-    - React-fishhook (= 1000.0.0)
-  - React-turbomodule-core (1000.0.0):
+    - React-cxxreact (= 0.60.0-microsoft.31)
+    - React-jsi (= 0.60.0-microsoft.31)
+  - React-jsinspector (0.60.0-microsoft.31)
+  - React-RCTActionSheet (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTAnimation (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTBlob (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+    - React-RCTNetwork (= 0.60.0-microsoft.31)
+    - React-RCTWebSocket (= 0.60.0-microsoft.31)
+  - React-RCTImage (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+    - React-RCTNetwork (= 0.60.0-microsoft.31)
+  - React-RCTLinking (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTNetwork (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTPushNotification (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTSettings (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTText (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTVibration (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+  - React-RCTWebSocket (0.60.0-microsoft.31):
+    - React-Core (= 0.60.0-microsoft.31)
+    - React-fishhook (= 0.60.0-microsoft.31)
+  - React-turbomodule-core (0.60.0-microsoft.31):
     - Folly (= 2018.10.22.00)
-    - React-Core (= 1000.0.0)
-    - React-cxxreact (= 1000.0.0)
-    - React-jsi (= 1000.0.0)
-    - React-turbomodule-core/core-ios (= 1000.0.0)
-  - React-turbomodule-core/core-ios (1000.0.0):
+    - React-Core (= 0.60.0-microsoft.31)
+    - React-cxxreact (= 0.60.0-microsoft.31)
+    - React-jscallinvoker (= 0.60.0-microsoft.31)
+    - React-jsi (= 0.60.0-microsoft.31)
+    - React-turbomodule-core/core-ios (= 0.60.0-microsoft.31)
+  - React-turbomodule-core/core-ios (0.60.0-microsoft.31):
     - Folly (= 2018.10.22.00)
-    - React-Core (= 1000.0.0)
-    - React-cxxreact (= 1000.0.0)
-    - React-jsi (= 1000.0.0)
-  - React-turbomodule-samples (1000.0.0):
+    - React-Core (= 0.60.0-microsoft.31)
+    - React-cxxreact (= 0.60.0-microsoft.31)
+    - React-jscallinvoker (= 0.60.0-microsoft.31)
+    - React-jsi (= 0.60.0-microsoft.31)
```

Comment:
Will these need to be bumped with every version release?

Reply:
The [upstream `master` `package.json` file](https://github.com/facebook/react-native/blob/d43059d0b801360c921bd1a3103274a179361925/package.json#L3) doesn't  have the current version number encoded into it, but rather is always at `1000`. Because we apparently _do_, this will automatically follow the version that trickles from the `package.json` into the various `podspec` files.

I figured I’d update this now until it’s clear to me why we _do_ have it in our `master` `package.json` and/or get rid of that.

Reply:
As discussed, we’ll leave this as-is for now–and deal with the fact that after the next release gets cut by CI a subsequent `pod install` will again change these artefacts–and then see if we can update these artefacts as part of the release process or another way that doesn’t require adding `pod install` to that process.